dask/dataframe/__init__.py

```diff
@@ -16,3 +16,7 @@
     from .io import read_parquet, to_parquet
 except ImportError:
     pass
+try:
+    from .core import isna
+except ImportError:
+    pass
```

Comment:
Missing a newline at the end of the file (usually this is something you can configure in your editor to prevent this).

Reply:
Have now configured my editor to automate this :). Thanks a lot for your guidance!